src/main/resources/META-INF/services/com.fasterxml.jackson.databind.Module

```diff
@@ -38,28 +38,3 @@ org.killbill.billing.util.tag.boilerplate.Module
 org.killbill.billing.overdue.api.boilerplate.Module
 org.killbill.billing.catalog.api.boilerplate.Module
 
-#  Generated by https://github.com/killbill/api-pojos. Do not edit!
-#  
-#  Copyright 2022-2022 The Billing Project, LLC
-#  
-#  The Billing Project licenses this file to you under the Apache License,
-#  version 2.0 (the "License"); you may not use this file except in compliance
-#  with the License.  You may obtain a copy of the License at:
-#  
-#     http://www.apache.org/licenses/LICENSE-2.0
-#  
-#  Unless required by applicable law or agreed to in writing, software
-#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
-#  License for the specific language governing permissions and limitations
-#  under the License.
-
-org.killbill.billing.payment.plugin.api.boilerplate.plugin.Module
```

Comment:
You should keep this one, as this is required. What I'm talking about is  these lines:

```
#  Generated by https://github.com/killbill/api-pojos. Do not edit!
#  
#  Copyright 2022-2022 The Billing Project, LLC
#  
#  The Billing Project licenses this file to you under the Apache License,
#  version 2.0 (the "License"); you may not use this file except in compliance
#  with the License.  You may obtain a copy of the License at:
#  
#     http://www.apache.org/licenses/LICENSE-2.0
#  
#  Unless required by applicable law or agreed to in writing, software
#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
#  License for the specific language governing permissions and limitations
#  under the License.
```

Reply:
Fixed